runtime/str.go

```diff
@@ -205,6 +205,36 @@ func strEq(f *Frame, v, w *Object) (*Object, *BaseException) {
 	return strCompare(v, w, False, True, False), nil
 }
 
+func strFind(f *Frame, args Args, _ KWArgs) (*Object, *BaseException) {
+	expectedTypes := []*Type{StrType, StrType, IntType, IntType}
```

Comment:
If args[1] is unicode then the CPython implementation dispatches to unicode.find(): https://github.com/python/cpython/blob/2.7/Objects/stringobject.c#L1724

If you'd rather not add unicode.find() to this PR, then please add a TODO.